_format/1.0/index.md

```diff
@@ -96,9 +96,56 @@ If a document does not contain a top-level `data` key, the `included` member
 
 The top-level [links object][links] **MAY** contain the following members:
 
-* `self`: the [link][links] that generated the current response document.
-* `related`: a [related resource link] when the primary data represents a
+* 1) Relative Links --> I've never enjoyed the complex relationship links that JSON spec provides. There are very few occurrences of other API titans implementing complex hateoas relationships. Some API's I've used provide detailed level links like invoices/{id}. I also found at SF, in an API consumer survey to a few hundred developers, that less than 5% of clients were activtely using the self-links. Most links were being built using env vars. After the lack of usage, it showcased how it wasn't really helpful. I just wanted to start the conversation and not necessarily provide the answer. I'm not sure how others feel on this topic, but these implementations can get REALLY painful through the application life cycle.
+
+* `urlName`: a [related resource link] when the primary data represents a
   resource relationship.
+  - We found the simplication of specific or explicit naming made it easier for the client to determine which URL to grab. We also found it even more consumer friendly returning paginated links as well.
+  - invoiceDetailUrl
+
+
+```json
+{
+  "data": [
+    {
+      "id": "1"
+    },
+    { "links": {
+        "invoiceDetailUrl": "full-context-root/invoices/1"
+      }
+    }
+  ]
+}
+```
+
+* 2) Flat payload --> I like the flat payload approach. Which is returning all the payload information inside the data object. The alternative is having your attributes under the attributes object.
+
+-- Before -- Title is nested under attributes.
```

Comment:
I'm confused about what you're proposing and how these JSON examples relate to links. The "before" example doesn't contain any links at all.

Reply:
I agree with Barrett here.

Reply:
Actually upon further reading, I think the docs have changed my opinion. It would be good to separate attributes from links and such, so I understand now.